.github/workflows/wheels.yml

```diff
@@ -210,13 +211,46 @@ jobs:
         run: |
           CIBW="pip; args: --no-build-isolation"
           echo "CIBW_BUILD_FRONTEND=$CIBW" >> "$GITHUB_ENV"
-
+      
       - name: Build wheels
+        if: matrix.buildplat[1] != 'win' || matrix.buildplat[2] != 'ARM64' # win_arm64 build configured separately
+        uses: pypa/cibuildwheel@cf078b0954f3fd08b8445a7bf2c3fb83ab3bb971 # v3.0.0b4
+        env:
+          CIBW_BUILD: ${{ matrix.python[0] }}-${{ matrix.buildplat[1] }}*
+          CIBW_ARCHS: ${{ matrix.buildplat[2] }}
+          CIBW_ENABLE: cpython-freethreading cpython-prerelease
+
+      # can be remove after https://github.com/actions/partner-runner-images/issues/90
+      - name: Install LLVM for win_arm64
+        if: matrix.buildplat[1] == 'win' && matrix.buildplat[2] == 'ARM64'
+        shell: pwsh
+        run: |
+          Invoke-WebRequest https://github.com/llvm/llvm-project/releases/download/llvmorg-20.1.6/LLVM-20.1.6-woa64.exe -UseBasicParsing -OutFile LLVM-woa64.exe
+          Start-Process -FilePath ".\LLVM-woa64.exe" -ArgumentList "/S" -Wait
+          echo "C:\Program Files\LLVM\bin" | Out-File -FilePath $env:GITHUB_PATH -Encoding utf8 -Append
+
+      # pkgconf and FortranRuntime hacks may be removed after fixes in according libraries
+      - name: Build wheels for win_arm64
+        if: matrix.buildplat[1] == 'win' && matrix.buildplat[2] == 'ARM64'
         uses: pypa/cibuildwheel@cf078b0954f3fd08b8445a7bf2c3fb83ab3bb971 # v3.0.0b4
         env:
           CIBW_BUILD: ${{ matrix.python[0] }}-${{ matrix.buildplat[1] }}*
           CIBW_ARCHS: ${{ matrix.buildplat[2] }}
           CIBW_ENABLE: cpython-freethreading cpython-prerelease
+          CIBW_ENVIRONMENT_WINDOWS: >
+            PATH="C:\\vcpkg\\installed\\arm64-windows\\tools\\pkgconf;$PATH"
+            CC=clang-cl
+            CXX=clang-cl
+            FC=flang
+            PKG_CONFIG_PATH=$GITHUB_WORKSPACE
+            TARGET_ARCH=${{ matrix.buildplat[2] }}
+          CIBW_BEFORE_BUILD_WINDOWS: >
+            pip install wheel delvewheel &&
+            pip install -r requirements/build.txt -r requirements/openblas.txt &&
```

Comment:
These installs aren't necessary, they're already done elsewhere, e.g. in `tools/wheels/cibw_before_build_win.sh`

Reply:
Let me try again, I think it was needed for some reason.